plugins/chat/assets/javascripts/discourse/components/chat-thread.hbs

```diff
@@ -2,18 +2,20 @@
   class={{concat-class "chat-thread" (if this.loading "loading")}}
   data-id={{this.thread.id}}
   {{did-insert this.loadMessages}}
-  {{did-update this.thread.id this.loadMessages}}
+  {{did-update this.loadMessages this.thread.id}}
```

Comment:
This is a kinda unexpected API from ember but yes this is function first and then the observed param

Reply:
Agh I keep messing this up, I really hate that its backwards like this. Thanks for fixing 👍